### PR TITLE
Use python clvm when needed

### PR DESCRIPTION
**Pull request by aqk**
Use the Python backend for features not yet supported in the rust backend

### CODE REVIEW

File: clvm_tools/cmds.py [hunk 1]
```diff
@@ -205,14 +205,15 @@ def launch_tool(args, tool_name, default_stage=0):
     try:
         output = "(didn't finish)"
 
-        if args.backend == "rust" or (serialize_and_run_program and args.backend != "python"):
+        use_rust = (tool_name != "run") and not args.table and not args.stage and (args.backend == "rust" or (serialize_and_run_program and args.backend != "python"))
```
Comment (richardkiss) [inline clvm_tools/cmds.py:208]: I bet `black` doesn't like this. (I'm not crazy about it either, but I know it's temporary).

File: tests/cmds_test.py [hunk 2]
```diff
@@ -89,6 +88,7 @@ def f(self):
                 f.write(actual_output)
                 f.close()
         self.assertEqual(expected_output, actual_output)
+
```
Comment (richardkiss) [inline tests/cmds_test.py:91]: ?

File: tests/cmds_test.py [hunk 3]
```diff
@@ -109,7 +109,7 @@ def inject(*paths):
 
 inject("cmd")
 
-#inject("v0_0_2")
+# inject("v0_0_2")
```
Comment (richardkiss) [inline tests/cmds_test.py:112]: ?

File: tests/cmds_test.py (hunk 2 above)
Comment (richardkiss) [reply tests/cmds_test.py:91]: `black`?

File: clvm_tools/cmds.py (hunk 1 above)
Comment (aqk) [reply clvm_tools/cmds.py:208]: I don't like it either, and I wrote it!

File: tests/cmds_test.py (hunk 2 above)
Comment (aqk) [reply tests/cmds_test.py:91]: I wonder if that was me or black/flake ...

File: tests/cmds_test.py (hunk 3 above)
Comment (aqk) [reply tests/cmds_test.py:112]: It's the formatter tools.

**Review by richardkiss — APPROVED** [review]
Happy to merge as is, or you can tweak the issues (might be worth letting `black` reformat that new long line).

**Pull request merged**